docs/getting_started/faq.rst

```diff
@@ -0,0 +1,98 @@
+Frequently Asked Questions (FAQs)
+=================================
+
+Below, you will find answer to the most commonly asked questions you may have about
+Modin. If you still cannot find the answer you are looking for, please post on 
+the #support channel on our Slack_ community or open a Github issue_.
+
+What’s the issue with pandas? Why should I use Modin?
+""""""""""""""""""""""""""""""""""""""""""""""""""""""
+In pandas, you are only able to use one core at a time when you are doing computation of
+any kind. This approach does not scale on larger data sets and adding more hardware does not
+solve the problem.  
+
+The ``modin.pandas`` `DataFrame`_ is an extremely light-weight parallel DataFrame. Modin
+transparently distributes the data and computation so that all you need to do is
+continue using the pandas API as you were before installing Modin. With Modin, 
+you are able to use all of the CPU cores on your machine, and because of it's light-weight
+nature often results in less memory overhead than pandas.
+
+How much faster can I go with Modin compared to pandas?
+""""""""""""""""""""""""""""""""""""""""""""""""""""""""
+Even in a traditionally serial task like ``read_csv``, we see large gains by efficiently 
+distributing the work across your entire machine. Because it is so light-weight, 
+Modin provides speed-ups of up to 4x on a laptop with 4 physical cores. This speedup scales
+efficiently with larger machines with more cores.
```

Comment:
In addition to these suggestions, we should add links to the Modin papers here.
```suggestion
How much faster can I go with Modin compared to pandas?
""""""""""""""""""""""""""""""""""""""""""""""""""""""""
Modin is designed to scale with the amount of hardware available.
Even in a traditionally serial task like ``read_csv``, we see large gains by efficiently 
distributing the work across your entire machine. Because it is so light-weight, 
Modin provides speed-ups of up to 4x on a laptop with 4 physical cores. This speedup scales
efficiently with larger machines with more cores. We have several published papers that
include performance results and comparisons against pandas.
```